### PR TITLE
Clarified users add workflow in the MP

### PR DESCRIPTION
**Pull request by alextopuzov**
- Fixes https://github.com/vmware/vic-product/issues/289
- Fixes https://github.com/vmware/vic-product/issues/337
- Fixes https://github.com/vmware/vic-product/issues/334

### CODE REVIEW

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md [hunk 1]
```diff
@@ -9,6 +9,6 @@ If you don't know the port number, you can access the portal by going to http://
 The role that has full permissions for vSphere Integrated Containers is the cloud administrator role. 
 By default, the cloud administrator role is assigned to the Administrators group for vCenter Server during the installation of vSphere Integrated Containers. Every user that is a member of that group in the Platform Services Controller can access the Management Portal as cloud administrator. After you log in as a cloud administrator, you can give other users access to vSphere Integrated Containers by assigning them roles in projects.
 
-Optionally, you can log in with a user that you created during the OVA deployment, if you used that option.
+Optionally, you can log in with an example user that you created during the OVA deployment, if you used that option.
```
Comment (stuclem) [inline docs/user_doc/vic_cloud_admin/logging_in_MP.md]: ... example user account...

File: docs/user_doc/vic_overview/introduction.md [hunk 2]
```diff
@@ -242,19 +242,29 @@ Role assigned per project. In addition to the view access, a developer can also:
...
 
-Role assigned per project.
-- For their projects, can add other users and assign other DevOps administrators.
+Role assigned per project. For their assigned projects DevOps administrators can perform additional actions:
+- Add developers and viewers and assign other DevOps administrators
+- Change the project configurations, such as making the project registry public, changing deployment security settings, and enabling vulnerability scanning
 
 **Cloud administrator / Global Administrator for all projects**
 
 The default cloud administrator role is assigned to the Administrators group for vCenter Server during the installation of vSphere Integrated Containers. Through the management portal, you can revoke that role for the Administrators group, only after you assign the role to another group. Cloud admins, can also assign the role to individual users.
```
Comment (stuclem) [inline docs/user_doc/vic_overview/introduction.md]: "Cloud admins, can" remove comma

File: docs/user_doc/vic_overview/introduction.md [hunk 3]
```diff
@@ -242,19 +242,29 @@ Role assigned per project. In addition to the view access, a developer can also:
...
 
 The following global permissions are unique for the cloud administrator role:
-- Add new cloud admins.
+- Add new cloud administrators
 - Create new projects and assign the first DevOps administrator for them
```
Comment (stuclem) [inline docs/user_doc/vic_overview/introduction.md]: "assign the first DevOps administrator for them" --> to them

File: docs/user_doc/vic_overview/introduction.md [hunk 4]
```diff
@@ -242,19 +242,29 @@ Role assigned per project. In addition to the view access, a developer can also:
...
 - Add hosts and clusters as resources to a given project
-- Add and manage registries
+- Add and manage registries, replication endpoints, and replication rules
+- Add predefined credentials and certificates for authentication
+- Set global configurations for registries handling
+- View system logs
+
+##### Example Users #####
 
+You can create Cloud administrator, DevOps administrator, and a Developer example users during the deployment of vSphere Integrated Containers features. These users create as local users for your Platform Services Controller. If you leave the default value for prefix for the users to be created, you end up with the following example users:
```
Comment (stuclem) [inline docs/user_doc/vic_overview/introduction.md]: "during the deployment of vSphere Integrated Containers features" --> "during the deployment of the vSphere Integrated Containers appliance"

File: docs/user_doc/vic_cloud_admin/logging_in_MP.md (hunk 1 above)
Comment (alextopuzov) [reply docs/user_doc/vic_cloud_admin/logging_in_MP.md]: fixed

File: docs/user_doc/vic_overview/introduction.md (hunk 2 above)
Comment (alextopuzov) [reply docs/user_doc/vic_overview/introduction.md]: fixed

File: docs/user_doc/vic_overview/introduction.md (hunk 3 above)
Comment (alextopuzov) [reply docs/user_doc/vic_overview/introduction.md]: fixed

File: docs/user_doc/vic_overview/introduction.md (hunk 4 above)
Comment (alextopuzov) [reply docs/user_doc/vic_overview/introduction.md]: lol fixed

File: docs/user_doc/vic_cloud_admin/add_cloud_admins.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+1. Go to http://<i>vic_appliance_address</i>, click the link to **Go to the vSphere Integrated Containers Management Portal**, and enter the vCenter Server Single Sign-On credentials.
+
+    Use an account with Cloud administrator privileges.
+2. Select **Administration** > **Identity Management**, and click **Users & Groups**.
+3. In the search box, enter a user name, email address, or the user's full name and press Enter.
+
+	Wait for the user to appear in the table.
+
+5. Select the check box next to the user in the table and click **Make Admin**.
```
Comment (sergiosagu) [inline docs/user_doc/vic_cloud_admin/add_cloud_admins.md]: Make Admin -> Assign Admin Role

Comment (alextopuzov) [reply docs/user_doc/vic_cloud_admin/add_cloud_admins.md]: image

Comment (alextopuzov) [reply docs/user_doc/vic_cloud_admin/add_cloud_admins.md]: I'd love to rewrite it but that's how you made the UI. :)

**Pull request merged**